processing/src/main/java/org/apache/druid/segment/data/CompressedVariableSizedBlobColumnSupplier.java

```diff
@@ -34,17 +34,29 @@ public class CompressedVariableSizedBlobColumnSupplier implements Supplier<Compr
   public static CompressedVariableSizedBlobColumnSupplier fromByteBuffer(
```

Comment:
This method seems like a bug waiting to happen, where someone _should_ provide both `compressionOrder` and `valueOrder` but only provides `compressionOrder` because this method exists. Remove it?